include/sys/zio_impl.h

```diff
@@ -73,9 +73,9 @@ extern "C" {
  * the supported transformations:
  *
  * Compression:
- * ZFS supports three different flavors of compression -- gzip, lzjb, and
- * zle. Compression occurs as part of the write pipeline and is performed
- * in the ZIO_STAGE_WRITE_BP_INIT stage.
+ * ZFS supports five different flavors of compression -- gzip, lzjb, lz4, zle,
```

Comment:
maybe such fixes should go to a separate PR

Reply:
why? its just documentation. and here zstd was added. so its part of the patch

Reply:
because it's a fix which is not related to zstd. let's see what the others say. maybe this is really over-particular ;)

Reply:
??? it just adds the line for zstd compression support. which is of course part of the patch. by the way. this documentation update was also taken from alans original bsd code

Reply:
* ZFS supports five different flavors of compression -- gzip, lzjb, lz4, zle,
 * and zstd. Compression occurs as part of the write pipeline and is
 * performed in the ZIO_STAGE_WRITE_BP_INIT stage.


so mentioning the zstd compression support is part of it